### PR TITLE
Gdr 2199

### PR DESCRIPTION
**Pull request by bczech**
# Description
## What changed?
Related JIRA issue: GDR-2199 and GDR-2244

## Why was it changed?
To simplify logic of combo data and rename `matrix` into `combination`

# Checklist for sustainable code base
- [ ] I added tests for any code changed/added
- [ ] I added documentation for any code changed/added
- [ ] I made sure naming of any new functions is self-explanatory and consistent

# Logistic checklist
- [X] Package version bumped
- [X] Changelog updated

# Screenshots (optional)

### CODE REVIEW

File: R/fit_SE.combinations.R [hunk 1]
```diff
@@ -398,50 +382,31 @@ fit_SE.combinations <- function(se,
...
-         metrics = metrics,
+    list(metrics = metrics,
          all_iso_points = all_iso_points,
          isobolograms = isobolograms,
-         smooth_mx = smooth_mx,
-         bliss_score = bliss_score,
-         hsa_score = hsa_score,
-         CIScore_50 = CIScore_50,
-         CIScore_80 = CIScore_80)
+         excess = excess,
```
Comment (gladkia) [inline R/fit_SE.combinations.R:390]: 💪

**Review by gladkia — APPROVED** [review]

**Review by marcin-kam — APPROVED** [review]

**Review by j-smola — APPROVED** [review]
Please, update package version
* gDRtestData (>= 1.1.3)
* gDRutils (>=1.1.3)
* gDRcomponents (>=1.4.36)
* gDRstyle(>=1.1.1)
* gDRimport(>=1.1.4)

File: R/combinations-calculate_matrix_metric.R [hunk 2]
```diff
@@ -34,12 +34,18 @@ NULL
...
 #' @export
 calculate_HSA <- function(sa1, series_id1, sa2, series_id2, metric) {
-  .calculate_matrix_metric(sa1, series_id1, sa2, series_id2, metric, FXN = pmin)
+  .calculate_matrix_metric(sa1,
+                           series_id1,
+                           sa2,
+                           series_id2,
+                           metric,
+                           FXN = pmin,
+                           measured_col = "smooth")
```
[the remark this answers is not in the document]
Comment (bczech) [reply R/combinations-calculate_matrix_metric.R:48]: done

File: R/combinations-calculate_matrix_metric.R [hunk 3]
```diff
@@ -50,7 +56,12 @@ calculate_HSA <- function(sa1, series_id1, sa2, series_id2, metric) {
 #' sa2 <- data.table::data.table(conc = rep(0, n), conc2 = seq(n), x = seq(n))
 #' calculate_Bliss(sa1, "conc", sa2, "conc2", "x")
```
[the remark this answers is not in the document]
Comment (bczech) [reply R/combinations-calculate_matrix_metric.R:57]: done

File: DESCRIPTION [hunk 4]
```diff
@@ -25,7 +25,11 @@ Imports:
     BiocParallel,
     checkmate,
     futile.logger,
+<<<<<<< HEAD
```
Comment (gladkia) [inline DESCRIPTION:28]: don't forget to clean DESCRIPTION

Comment (bczech) [reply DESCRIPTION]: Done

**Pull request merged**